blog/markdown-fidelity.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Flowershow Supports Full CommonMark + GitHub Flavored Markdown
+authors:
+  - olayway
+date: 2025-11-04
+---
+
+When we created **Flowershow**, our mission was simple:
```

Comment:
Lead with the headline (what you have next). What's new / important today? e.g. what you have next

> **Flowershow now supports full CommonMark + GitHub-Flavored Markdown spec.**

And is there a way to make this a bit sexier / clearer to average user (who may not even know what Commonmark etc is). Btw didn't we support all commonmark before? (or maybe not because of MDX stuff?).